db/migrate/20191126141806_create_events.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateEvents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :events do |t|
+      t.string :title
+      t.string :location
+      t.integer :user_id
+      t.datetime :date
```

Comment:
- [ ] We should have a foreign key to represent the [creator](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) of the event